routing/notifications_test.go

```diff
@@ -511,6 +526,8 @@ func TestEdgeUpdateNotification(t *testing.T) {
 				"expected %v, got %v", edgeAnn.TimeLockDelta,
 				edgeUpdate.TimeLockDelta)
 		}
+		require.Equal(t, edgeAnn.ExtraOpaqueData,
```

Comment:
nit: format like other multi-line function calls.

Reply:
That line came from me, we don't do this very consistently over the code base for `require`. I was under the impression that this falls into the logging-style line breaking (I was told about that somewhere, but can't remember). Perhaps we could clarify this in the style guide

Reply:
I think this is only true if it's a call with a format string, so in this example `require.Equalf()` instead of `require.Equal()`. But yes, more clarity on it certainly doesn't hurt.